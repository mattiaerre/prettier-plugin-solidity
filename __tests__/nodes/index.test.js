const nodes = require('../../src/nodes');

test('nodes list to match snapshot', () => {
  const list = Object.keys(nodes).reduce((accumulator, current) => {
    accumulator.push(current);
    return accumulator;
  }, []);

  expect(list).toMatchSnapshot();
});
