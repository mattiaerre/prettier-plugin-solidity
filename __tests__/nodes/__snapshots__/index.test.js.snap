// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nodes list to match snapshot 1`] = `
Array [
  "ContractDefinition",
  "ImportDirective",
  "PragmaDirective",
  "SourceUnit",
]
`;
